app.js

```diff
@@ -0,0 +1,44 @@
+const { request } = require("http");
+
+class App {
+  commands = [];
+
+  registerCommand(name, callback) {
+    const object = { name: name, callback: callback };
+    this.commands.push(object);
+  }
+
+  showCommands() {
+    const names = this.commands.map((item) => item.name);
+    return "Привет, я чат-бот! Список моих команд: " + names.join(", ") + " :)";
+  }
+
+  listenToCommand() {
+    const readline = require("readline");
+    const rl = readline.createInterface({
+      input: process.stdin,
+      output: process.stdout,
+    });
+
+    rl.question("Введите команду: ", (userCommand) => {
+      const command1 = this.commands.find(item => item.name === userCommand);
```

Comment:
Почему такое странное имя? Когда я подсказывал, я взял рандомное имя. В коде переменные всегда должны иметь корректные имена. Например `command`, или `currentCommand`.
```suggestion
      const command = this.commands.find(item => item.name === userCommand);
```